src/Microsoft.DotNet.XHarness.CLI/Commands/Apple/Simulators/SimulatorsCommand.cs

```diff
@@ -393,23 +395,33 @@ private async Task<(string XcodeVersion, string XcodeUuid)> GetXcodeInformation(
 
         xcodeVersion = xcodeVersion.Trim();
 
-        string xcodeUuid;
+        xcodeVersion = xcodeVersion.Insert(xcodeVersion.Length - 2, ".");
```

Comment:
Can we have a comment here explaining why?

Reply:
added the comment